src/scenes/CreateEvent/index.js

```diff
@@ -193,66 +285,51 @@ class CreateEvent extends React.Component {
       eventEscrowAmount,
       creatorAddress,
     );
-
     this.props.reset(FORM_NAME);
-  };
-
-  renderTextField = ({
-    input,
-    placeholder,
-    startAdornmentLabel,
-    meta: { touched, error },
-    ...custom
-  }) => (
-    <FormControl fullWidth>
-      <TextField
-        {...input}
-        {...custom}
-        fullWidth
-        placeholder={placeholder}
-        error={Boolean(touched && error)}
-        InputProps={{
-          startAdornment: startAdornmentLabel ? <InputAdornment position="start">{startAdornmentLabel}</InputAdornment> : null,
-        }}
-      />
-      {
-        touched && error ?
-          <FormHelperText error>{error}</FormHelperText> : null
-      }
-    </FormControl>
-  );
+  }
 
   onSelectResultSetterAddress = () => {
     this.setState({
       selectAddressDialogVisibility: true,
     });
-  };
+  }
 
   onSelectResultSetterAddressDialogClosed = (address) => {
     this.setState({
       selectAddressDialogVisibility: false,
     });
 
     this.props.changeFormFieldValue(ID_RESULT_SETTER, address);
-  };
+  }
 
   onClose = () => {
     this.props.toggleCreateEventDialog(false);
-  };
+  }
+
+  validateEnoughQTUM = () => {
+    const totalQtum = _.sumBy(this.props.walletAddresses, ({ qtum }) => qtum ? qtum : 0); // eslint-disable-line
```

Comment:
can we do this without disabling the eslint?

Reply:
yep! Just tested in dev console on lodash site.
```js
var list = [
    {
		qtum: 1
    },
    {
		qtum: 3
    },
]
var totalQTUM = _.sumBy(list, ({ qtum }) => qtum)
console.log('TOTAL: ', totalQTUM)
TOTAL:  4
```

Reply:
cool. I just feel like we should obey the preset ESLint rules unless we have a good reason to disable a rule, which is not just for convenience's sake.